src/features/createSurvey/AddSurveyForm/selectionTextfieldList/SelectionTextfieldList.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { css } from '@emotion/react';
+
+import SelectionTextfield from '~/features/createSurvey/AddSurveyForm/selectionTextfieldList/SelectionTextfield';
+
+interface Props {
+  inputs: string[];
+  basicCount: number;
+
+  setInputs: (inputs: string[]) => void;
```

Comment:
```suggestion
  setInputs: Dispatch<SetStateAction<string[]>>;
```

string[] 타입의 state의 set 함수를 요구하시길 원하신다면 이렇게 바꾸면 어떨까요 ??

Reply:
이 방식도 알고 있긴 했는데, 이게 나을까요..? 
일단 변경하도록 할게요!